pkg/stash/with_azureblob.go

```diff
@@ -142,9 +93,10 @@ func (s *azblobLock) acquireLease(ctx context.Context, blobURL azblob.BlockBlobU
 	if err != nil {
 		return "", errors.E(op, err)
 	}
+	ttlSeconds := int32(ttl / time.Second)
 	for {
-		// acquire lease for 15 sec (it's the min value)
-		res, err := blobURL.AcquireLease(tctx, leaseID.String(), 15, azblob.ModifiedAccessConditions{})
+		// acquire lease for 20 sec
```

Comment:
Given that ttl is passed by the caller, we can't know for sure it's 20 seconds. Also given the previous comment that 15 seconds is the minimum, it might be worth checking whether azure will return an error or whether it implicitly ensures it is 15 seconds. If the former, we wanna make sure we bump it so we don't error out.